spec/features/pairings_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+feature "pairings" do
+
+  context "authorized user" do
+    let(:user) { FactoryGirl.create(:user) }
+
+    scenario "clicks pair in dashboard" do
+      partner = FactoryGirl.create(:user)
+      sign_in_as(user)
+
+      visit dashboard_path
+      expect(page).to have_content('Paired?')
+
+      find(:css, "##{partner.id}").click
+
+      expect(page).to have_content('Complete')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.